### PR TITLE
docs: add details to webhook set up instructions

### PR DESCRIPTION
**Pull request by askervin**

### CODE REVIEW

**Review by klihub — COMMENTED** [review]
LGTM. @marquiz WDYT ?

File: docs/webhook.md [hunk 1]
```diff
@@ -14,11 +14,41 @@ resource*s.
...
 Webhook](/cmd/cri-resmgr-webhook). Once you built the docker image for it using
-the [provided Dockerfile][/cmd/cri-resmgr-webhook/Dockerfile] and published it,
-you can set up the webhook with these commands:
-
-```
+the [provided Dockerfile](/cmd/cri-resmgr-webhook/Dockerfile) and published it,
+you can set up the webhook as follows:
+- Fill in the `IMAGE_PLACEHOLDER` in [webhook-deployment.yaml](/cmd/cri-resmgr-webhook/webhook-deployment.yaml) to match the image.
+- Create a `cri-resmgr-webhook-secret` that carries a key and a certificate to `cri-resmgr-webhook`. You can create a key, a self-signed certificate and the secret that holds them with commands:
+  ```
```
Comment (marquiz) [inline docs/webhook.md:21]: nit: I think we should start using language definitions in code blocks. i.e.` ```bash`  here in order to get nicer syntax-highlighted output.

File: docs/webhook.md [hunk 2]
```diff
@@ -14,11 +14,41 @@ resource*s.
...
+    svc.key: $(base64 -w0 < cmd/cri-resmgr-webhook/server-key.pem)
+  type: Opaque
+  EOF
+  kubectl create namespace $NS
+  kubectl create -f cmd/cri-resmgr-webhook/webhook-secret.yaml
+  ```
+- Fill in the `CA_BUNDLE_PLACEHOLDER` in [mutating-webhook-config.yaml](/cmd/cri-resmgr-webhook/mutating-webhook-config.yaml).
+  If you created the key and the certificate with the commands above,
+  you can do this with command:
+  ```
```
Comment (marquiz) [inline docs/webhook.md:45]: ` ```bash`

File: docs/webhook.md [hunk 3]
```diff
@@ -14,11 +14,41 @@ resource*s.
...
+  sed -e "s/CA_BUNDLE_PLACEHOLDER/$(base64 -w0 < cmd/cri-resmgr-webhook/server-crt.pem)/" \
+      -i cmd/cri-resmgr-webhook/mutating-webhook-config.yaml
+  ```
+- Finally set up the webhook with these commands:
+  ```
```
Comment (marquiz) [inline docs/webhook.md:50]: We could convert this to ` ```bash` too

**Comment by askervin** [comment]
I hope the generated docs have better bash syntax highlighting than github viewer, which highlighted only redirections and EOF (good) and the wait in kubectl wait (bad).

But at least now we've given them a chance.

**Review by marquiz — APPROVED** [review]
Thanks @askervin ! Let's remove the language definitions if highlighting looks too bad
LGTM

**Comment by klihub** [comment]
@askervin Do you want to keep the bash-specific highlighting despite its the results ? If yes, I'll merge this in its current state.

**Comment by askervin** [comment]
Yeah, let's go with `bash` highlights.

**Pull request merged**